src/components/ReviewList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+import { fetchReviews } from '../api';
+import '../styles/ReviewList.css';
```

Comment:
Getting an error from this because it should be import '../styles/reviewList.css'. - lowercase r :)